packages/backend/server/src/core/workspaces/controller.ts

```diff
@@ -109,11 +109,6 @@ export class WorkspacesController {
     }
 
     res.setHeader('content-type', 'application/octet-stream');
-    res.setHeader(
-      'last-modified',
-      new Date(binResponse.timestamp).toUTCString()
-    );
-    res.setHeader('cache-control', 'private, max-age=2592000');
```

Comment:
publish mode means our blog's update?
i think it can be adjust in cloudflare, its a specific scenario
in normal share pages we should tend to use caching

Reply:
shared docs